AngularLanguageService.Shared/LanguageServer/MiddleLayer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.LanguageServer.Client;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+using Newtonsoft.Json.Linq;
+
+namespace AngularLanguageService.LanguageServer
+{
+	[Export]
+	internal sealed class MiddleLayer : ILanguageClientMiddleLayer
+	{
+		#region ILanguageClientMiddleLayer
+		bool ILanguageClientMiddleLayer.CanHandle(string methodName) => true;
+
+		Task ILanguageClientMiddleLayer.HandleNotificationAsync(string methodName, JToken methodParam, Func<JToken, Task> sendNotification)
+			=> sendNotification(methodParam);
+
+		Task<JToken?> ILanguageClientMiddleLayer.HandleRequestAsync(string methodName, JToken methodParam, Func<JToken, Task<JToken?>> sendRequest)
+		{
+			if (string.Equals(methodName, Methods.TextDocumentCompletionName))
+			{
+				return Task.FromResult<JToken?>(null);
+			}
+			else
+			{
+				return sendRequest(methodParam);
```

Comment:
The previous version had some logging (to output pane) that I found useful at least for debugging. That being said, maybe LSP already has logging that they show somewhere else that I don't know about?

Reply:
Yeah AppData\Local\Temp\VSLogs has information about the LSP communication, and together with the verbose settings we're using the Angular server, it feels enough IMO.